src/controllers/attachment/index.ts

```diff
@@ -87,25 +95,23 @@ export class attachment extends Controller {
     @Request() req: express.Request,
     @Path() id: string,
     @Header('return-type') type: 'json' | 'file'
-  ): Promise<Attachment> {
+  ): Promise<Attachment | Blob> {
```

Comment:
This doesn't return an `Attachment | Blob` it returns `any | Blob`

Reply:
Thanks for a flag up, indeed we are returning whatever JSON is! Updated to be an object since we use JSON.parse() it has to be an `JSON`, right? Happy to change to `<any>`